Transformation.cs

```diff
@@ -16,7 +14,7 @@ namespace Linq2
     // extra parameter of type TransformationFunction, and
     // return an IEnumerable<int>.
 
-    public static class naser
+    public static class Transformation
     {
         public static IEnumerable<int> Transform(this IEnumerable<int> Li, TransformationFunction number)
```

Comment:
use readable names don't be lazy 😅.
e.g : Li => items, and number => transformer

Reply:
hhhhhhhhhh
Done ...